examples/resources/cockroach_cluster/resource.tf

```diff
@@ -30,3 +31,16 @@ resource "cockroach_cluster" "serverless" {
   ]
   delete_protection = false
 }
+
+resource "cockroach_cluster" "basic" {
```

Comment:
Here's what is "basic" and what is "cockroach-basic"?

Reply:
See above: "basic" is the name of the TF resource. "cockroach-basic" is the name of the CC cluster.